### PR TITLE
Added Home-page in Video Library

### PR DESCRIPTION
**Pull request by nikhilmalviya-dev**
in video Library i added Home Page.
and in home page i added Navigation bar , genre etc
disney
.

### CODE REVIEW

File: src/Componant/Genre/genre.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.cateImg{
```
Comment (Priya-Patel31) [inline src/Componant/Genre/genre.css:1]: fix : could have used better naming like categoryImage .

File: src/Componant/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <section>
+        <input
+          type="search"
+          placeholder="search songs"
+          className="Serch-box padding-8p font-16p"
+        />
+      </section>
+      <section>
+        <Link to="/login">
+          <i class="fas fa-user loginIcon font-26p hoverText"></i>
```
Comment (Priya-Patel31) [inline src/Componant/Navbar/Navbar.jsx]: Suggestion : can use react icons

File: src/Componant/Navbar/navbar.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    border-bottom:2px solid var(--purple-700);
+    font-family: 'Libre Bodoni', serif;
+    z-index: 1;
+}
+
+.brand-logo{
+    width: 8.53rem;
+}
+
+.Serch-box{
```
Comment (Priya-Patel31) [inline src/Componant/Navbar/navbar.css]: fix : search spelling is wrong

File: src/index.js [hunk 4]
```diff
@@ -3,13 +3,22 @@ import ReactDOM from "react-dom";
...
+import {BrowserRouter} from "react-router-dom";
+import {FilterProvider} from "./Context/Filter-context";
+import {GenreProvider} from "./Context/GenreContext";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
+  <BrowserRouter>
```
Comment (Priya-Patel31) [inline src/index.js]: use proper indentation.

File: src/Componant/Genre/Genre.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+const Genre = () => {
+  const { genre } = useGenre();
+  const { VideoDispatch } = useFilter();
+
+  return (
+    <main className="genre-container flex-justify-between flex-wrap gap-1r margin-1r">
+    
+      {genre.map((item) => (
+        <section className="genre-contain flex-column align-center gap-8p padding-4px ">
```
Comment (Priya-Patel31) [inline src/Componant/Genre/Genre.jsx:14]: fix : provide key to it.

**Review by Priya-Patel31 — APPROVED** [review]
Code is well formatted . Just suggested few changes please go through it  and make those changes . Do provide a preview link for checking the functionality of the app .

File: src/Componant/Genre/genre.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    width:16.5rem;
+    height:29vh; 
+    border-radius: 0.5rem; 
+}
+
+.genre-contain{
+    color:var(--white-200);
+    box-shadow:2px 2px  2px rgb(0 6 9);
```
Comment (Bharatrathod-81) [inline src/Componant/Genre/genre.css:9]: you can use var color instead of hard coded color

**Review by Bharatrathod-81 — APPROVED** [review]

**Pull request merged**